src/LndClient/Data/OpenChannel.hs

```diff
@@ -0,0 +1,58 @@
+{-# LANGUAGE DeriveGeneric #-}
+
+module LndClient.Data.OpenChannel
+  ( OpenChannelRequest (..),
+    ChannelPoint (..),
+    ResultWrapper (..),
+  )
+where
+
+import Data.Aeson (FromJSON (..), ToJSON (..))
+import Data.Text (Text)
+import GHC.Generics (Generic)
+import LndClient.Utils (stdParseJSON, stdToJSON)
+
+newtype ResultWrapper a
+  = ResultWrapper
+      { result :: a
+      }
+  deriving (Generic, Show)
+
+instance (FromJSON a) => FromJSON (ResultWrapper a) where
+  parseJSON = stdParseJSON
+
+data OpenChannelRequest
+  = OpenChannelRequest
+      { nodePubkey :: Text,
+        localFundingAmount :: Text,
+        pushSat :: Text,
```

Comment:
I'm pretty sure "localFundingAmount" and "pushSat" should be "MoneyAmount" type, not "Text"

Reply:
In this case it should be encoded to json request as text. Is it possible?